app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    file_name = command_list[1]
+    file_copy_name = command_list[2]
+
+    if file_name == file_copy_name:
+        return
+
+    with (open(file_copy_name, "r") as file_in,
```

Comment:
file_copy_name as read????
Did you test your code?

Reply:
Actually, no),  I haven't read checklist, my bad...